java/cdk/pom.xml

```diff
@@ -50,6 +50,11 @@
             <artifactId>bulk-import-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>sleeper</groupId>
+            <artifactId>query-core</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
```

Comment:
It doesn't look like this dependency is used?

Reply:
Removed